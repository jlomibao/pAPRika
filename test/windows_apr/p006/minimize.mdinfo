

   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.1271E+02     4.7171E-01     1.6114E+00     H4         17

 BOND    =        1.1396  ANGLE   =      107.7354  DIHED      =       51.6190
 VDWAALS =      -50.8788  EEL     =     1336.3742  EGB        =     -132.0678
 1-4 VDW =        5.2524  1-4 EEL =    -2131.8979  RESTRAINT  =        0.0165
 EAMBER  =     -812.7239
 NMR restraints: Bond =    0.012   Angle =     0.003   Torsion =     0.000
===============================================================================
