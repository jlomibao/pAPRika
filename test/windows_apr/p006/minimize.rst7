defa
  125
  -0.5218531   4.9768316   1.8322080   1.0158387   4.9119771   1.8321273
  -0.8854056   4.2744775   0.6087464  -0.8853946   4.2884600   3.0640476
  -0.9013282   6.0051334   1.8266644   1.4806150   5.9046439   1.8264496
   1.3192367   4.1812129   0.6089874   1.3204542   4.1954726   3.0637458
   2.6855683   4.0816166   0.1123550   2.6885427   4.0866585   3.5558651
   0.2026000   3.8854390   3.7927317   0.2010407   3.8509487  -0.1106865
   0.1830781   3.4066772   4.9127524   0.1798122   3.3375305  -1.2162941
   2.6609808   4.0447895  -0.9828535   3.2440692   4.9644575   0.4441740
   2.6663873   4.0489354   4.6510950   3.2504059   4.9673214   3.2239347
  -4.6341084   1.9602187   1.8340387  -3.8083602   3.2587436   1.8338702
  -4.2129989   1.2966201   0.6072273  -4.2127564   1.2962136   3.0605242
  -5.7143982   2.1457303   1.8341870  -4.4345881   4.1583432   1.8337801
  -3.0292923   3.1582901   0.6068831  -3.0278742   3.1594279   3.0603141
  -2.2552404   4.2892583   0.1120655  -2.2577571   4.2940915   3.5564755
  -3.3062045   2.0281406   3.7824936  -3.3039261   2.0256061  -0.1132564
  -2.8801084   1.7577797   4.8918066  -2.8710061   1.7506147  -1.2193163
  -2.2339333   4.2499276  -0.9831246  -2.7380918   5.2157443   0.4435973
  -2.2386822   4.2538627   4.6516742  -2.7440780   5.2188546   3.2252047
  -4.0843823  -3.1101942   1.8321545  -4.7958923  -1.7453639   1.8322444
  -3.2956828  -3.0754187   0.6077192  -3.3036567  -3.0800262   3.0620131
  -4.7855658  -3.9526701   1.8285749  -5.8880755  -1.8378900   1.8288427
  -4.3159858  -1.1188483   0.6076780  -4.3239111  -1.1223736   3.0620356
  -4.9096660   0.1163203   0.1122008  -4.9123376   0.1167312   3.5559910
  -3.4919831  -1.9335029   3.7877836  -3.4715454  -1.9228428  -0.1119888
  -3.0581778  -1.7069442   4.9033343  -3.0159098  -1.6848219  -1.2175018
  -4.8654929   0.1142651  -0.9830101  -5.9533372   0.1609710   0.4441524
  -4.8674152   0.1155325   4.6511727  -5.9562225   0.1641548   3.2250976
   0.5847576  -5.1634297   1.8343025  -0.9527025  -5.0983290   1.8340704
   0.9484995  -4.4668053   0.6074232   0.9486655  -4.4659269   3.0606483
   0.9648517  -6.1915198   1.8346259  -1.4183059  -6.0906244   1.8342157
  -1.2558055  -4.3734403   0.6071109  -1.2569250  -4.3725459   3.0602910
  -2.6225046  -4.2691926   0.1122440  -2.6245378  -4.2718697   3.5555702
  -0.1384135  -4.0456004   3.7818046  -0.1376350  -4.0445347  -0.1130529
  -0.1175008  -3.5381450   4.8897628  -0.1161704  -3.5321202  -1.2191182
  -2.5990278  -4.2318723  -0.9829721  -3.1831267  -5.1506020   0.4442916
  -2.5999936  -4.2348349   4.6507726  -3.1828254  -5.1549707   3.2240843
   4.6983690  -2.1486432   1.8331827   3.8732900  -3.4478824   1.8331432
   4.2743597  -1.4839846   0.6080610   4.2798617  -1.4870019   3.0621152
   5.7787280  -2.3337236   1.8306792   4.5001552  -4.3470320   1.8305407
   3.0913386  -3.3466722   0.6080287   3.0967066  -3.3504678   3.0621858
   2.3191482  -4.4789641   0.1125829   2.3197056  -4.4806509   3.5562218
   3.3796774  -2.2227405   3.7868266   3.3649854  -2.2134284  -0.1118790
   2.9624893  -1.9583116   4.9005516   2.9313616  -1.9384874  -1.2176838
   2.2988857  -4.4397415  -0.9826309   2.8029258  -5.4048223   0.4445182
   2.2983380  -4.4412916   4.6514097   2.8016775  -5.4077361   3.2250854
   4.1471236   2.9225862   1.8338833   4.8563145   1.5567503   1.8343050
   3.3633012   2.8899298   0.6061522   3.3615495   2.8914291   3.0607707
   4.8495757   3.7640122   1.8338155   5.9486860   1.6471106   1.8347364
   4.3791405   0.9318269   0.6075978   4.3780997   0.9318193   3.0605552
   4.9714657  -0.3041504   0.1123460   4.9727532  -0.3035371   3.5566093
   3.5372948   1.7390643   3.7817288   3.5364749   1.7366856  -0.1128752
   3.0865917   1.5045178   4.8895259   3.0802734   1.4986512  -1.2181500
   4.9267214  -0.3024150  -0.9828423   6.0153325  -0.3492986   0.4436148
   4.9272128  -0.3008852   4.6517629   6.0170438  -0.3464871   3.2263857
   0.0007072  -0.0175670   6.2987717  -0.9127006  -0.6004353   6.1280964
   0.8637905  -0.6721963   6.1263448   0.0326644   0.7895932   5.5567111
   0.0245300   0.5535281   7.7298093   0.9321450   1.1549379   7.8629911
  -0.0221513  -0.5618067   8.7969750  -0.8300428   1.2282114   7.8628653
  -0.9294227  -1.1629068   8.6601004   0.8319646  -1.2363818   8.6604610
   0.0008073  -0.0016281  10.2327060  -0.8620594   0.6515458  10.4116364
  -0.0333729  -0.8183467  10.9641378   0.9147862   0.5777269  10.4120994
   0.0002181  -0.0008200  -6.0050177  -0.0004926   0.0001336  -9.0000703
   0.0002547   2.2000034 -11.2000237
