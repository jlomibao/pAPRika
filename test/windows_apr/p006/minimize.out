
          -------------------------------------------------------
          Amber 18 SANDER                              2018
          -------------------------------------------------------

| Run on 04/10/2019 at 18:22:08

|   Executable path: sander
| Working directory: /home/jlomibao/pAPRika_fork/pAPRika/test/windows_apr/p006
|          Hostname: entropy.ucsd.edu

  [-O]verwriting output

File Assignments:
|  MDIN: minimize.in                                                           
| MDOUT: minimize.out                                                          
|INPCRD: cb6-but-dum.rst7                                                      
|  PARM: cb6-but-dum.prmtop                                                    
|RESTRT: minimize.rst7                                                         
|  REFC: cb6-but-dum.rst7                                                      
| MDVEL: mdvel                                                                 
| MDFRC: mdfrc                                                                 
|  MDEN: minimize.mden                                                         
| MDCRD: minimize.nc                                                           
|MDINFO: minimize.mdinfo                                                       
|  MTMD: mtmd                                                                  
|INPDIP: inpdip                                                                
|RSTDIP: rstdip                                                                
|INPTRA: inptraj                                                               


 Here is the input file:

PBC MD Simulation                                                              
 &cntrl                                                                        
  imin =          1,                                                           
  ntx =           1,                                                           
  irest =         0,                                                           
  maxcyc =        5000,                                                        
  ncyc =          5000,                                                        
  dt =            0.002,                                                       
  nstlim =        5000,                                                        
  ntpr =          500,                                                         
  ntwe =          500,                                                         
  ntwr =          5000,                                                        
  ntwx =          500,                                                         
  ntxo =          1,                                                           
  ioutfm =        1,                                                           
  ntf =           2,                                                           
  ntc =           2,                                                           
  cut =           999.0,                                                       
  igb =           1,                                                           
  tempi =         298.15,                                                      
  temp0 =         298.15,                                                      
  ntt =           3,                                                           
  gamma_ln =      1.0,                                                         
  ig =            -1,                                                          
  ntp =           0,                                                           
  barostat =      2,                                                           
  ntr =           1,                                                           
  restraint_wt =  50.0,                                                        
  restraintmask = '@DUM',                                                      
  nmropt =        1,                                                           
  pencut =        -1,                                                          
  ntb =           0,                                                           
 /                                                                             
 &wt type = 'END', /                                                           
DISANG = rest.in                                                               
LISTOUT = POUT                                                                 
                                                                               

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

| Flags:                                                                        
| New format PARM file being parsed.
| Version =    1.000 Date = 01/16/19 Time = 17:42:13
 NATOM  =     125 NTYPES =       7 NBONH =      46 MBONA  =      93
 NTHETH =     118 MTHETA =     158 NPHIH =     200 MPHIA  =     375
 NHPARM =       0 NPARM  =       0 NNB   =     836 NRES   =       5
 NBONA  =      93 NTHETA =     158 NPHIA =     375 NUMBND =       6
 NUMANG =      12 NPTRA  =      13 NATYP =       7 NPHB   =       0
 IFBOX  =       0 NMXRS  =     108 IFCAP =       0 NEXTRA =       0
 NCOPY  =       0

 Implicit solvent radii are modified Bondi radii (mbondi)                                                   

|     Memory Use     Allocated
|     Real               14705
|     Hollerith            382
|     Integer            27997
|     Max Pairs              1
|     nblistReal             0
|     nblist Int             0
|       Total              225 kbytes

| Note: 1-4 EEL scale factors are being read from the topology file.

| Note: 1-4 VDW scale factors are being read from the topology file.
| Duplicated    0 dihedrals
| Duplicated    0 dihedrals
Note: ig = -1. Setting random seed to   613444 based on wallclock time in microseconds.

--------------------------------------------------------------------------------
   2.  CONTROL  DATA  FOR  THE  RUN
--------------------------------------------------------------------------------

default_name                                                                    

General flags:
     imin    =       1, nmropt  =       1

Nature and format of input:
     ntx     =       1, irest   =       0, ntrx    =       1

Nature and format of output:
     ntxo    =       1, ntpr    =     500, ntrx    =       1, ntwr    =    5000
     iwrap   =       0, ntwx    =     500, ntwv    =       0, ntwe    =     500
     ioutfm  =       1, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       2, ntb     =       0, igb     =       1, nsnb    =      25
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     = 999.00000, intdiel =   1.00000
     saltcon =   0.00000, offset  =   0.09000, gbalpha=    1.00000
     gbbeta  =   0.00000, gbgamma =   0.00000, surften =   0.00500
     rdt     =   0.00000, rgbmax  =  25.00000  extdiel =  78.50000
     alpb  =        0

Frozen or restrained atoms:
     ibelly  =       0, ntr     =       1
     restraint_wt =  50.00000

Energy minimization:
     maxcyc  =    5000, ncyc    =    5000, ntmin   =       1
     dx0     =   0.01000, drms    =   0.00010

SHAKE:
     ntc     =       2, jfastw  =       0
     tol     =   0.00001

NMR refinement options:
     iscale  =       0, noeskp  =       1, ipnlty  =       1, mxsub   =       1
     scalm   = 100.00000, pencut  =  -1.00000, tausw   =   0.10000
|  INFO: Old style inpcrd file read


    LOADING THE CONSTRAINED ATOMS AS GROUPS


   5.  REFERENCE ATOM COORDINATES

      
     Mask @DUM; matches     3 atoms

--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

                                                                                
 begin time read from input coords =     0.000 ps



           Begin reading energy term weight changes/NMR restraints
 WEIGHT CHANGES:
                         ** No weight changes given **

 RESTRAINTS:
 Requested file redirections:
  DISANG    = rest.in
  LISTOUT   = POUT
 Restraints will be read from file: rest.in
Here are comments from the DISANG input file:

                       Number of restraints read =     9

                  Done reading weight changes/NMR restraints


 Number of triangulated 3-point waters found:        0

--------------------------------------------------------------------------------
   4.  RESULTS
--------------------------------------------------------------------------------



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
      1      -7.9858E+02     1.0086E+01     2.4849E+01     N3          8

 BOND    =       11.0328  ANGLE   =      109.3438  DIHED      =       52.9822
 VDWAALS =      -50.3294  EEL     =     1326.5886  EGB        =     -126.5404
 1-4 VDW =        5.8710  1-4 EEL =    -2127.5276  RESTRAINT  =        0.0000
 EAMBER  =     -798.5791
 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    500      -8.1271E+02     4.7174E-01     1.6114E+00     H4         17

 BOND    =        1.1389  ANGLE   =      107.7351  DIHED      =       51.6196
 VDWAALS =      -50.8769  EEL     =     1336.3572  EGB        =     -132.0639
 1-4 VDW =        5.2513  1-4 EEL =    -2131.8852  RESTRAINT  =        0.0164
 EAMBER  =     -812.7239
 NMR restraints: Bond =    0.012   Angle =     0.003   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1000      -8.1271E+02     4.7174E-01     1.6114E+00     H4         17

 BOND    =        1.1390  ANGLE   =      107.7351  DIHED      =       51.6195
 VDWAALS =      -50.8771  EEL     =     1336.3591  EGB        =     -132.0644
 1-4 VDW =        5.2515  1-4 EEL =    -2131.8867  RESTRAINT  =        0.0164
 EAMBER  =     -812.7239
 NMR restraints: Bond =    0.012   Angle =     0.003   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1500      -8.1271E+02     4.7174E-01     1.6114E+00     H4         17

 BOND    =        1.1391  ANGLE   =      107.7352  DIHED      =       51.6194
 VDWAALS =      -50.8773  EEL     =     1336.3611  EGB        =     -132.0648
 1-4 VDW =        5.2516  1-4 EEL =    -2131.8881  RESTRAINT  =        0.0164
 EAMBER  =     -812.7239
 NMR restraints: Bond =    0.012   Angle =     0.003   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2000      -8.1271E+02     4.7174E-01     1.6114E+00     H4         17

 BOND    =        1.1392  ANGLE   =      107.7352  DIHED      =       51.6194
 VDWAALS =      -50.8776  EEL     =     1336.3628  EGB        =     -132.0652
 1-4 VDW =        5.2517  1-4 EEL =    -2131.8892  RESTRAINT  =        0.0164
 EAMBER  =     -812.7238
 NMR restraints: Bond =    0.012   Angle =     0.003   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2500      -8.1271E+02     4.7173E-01     1.6114E+00     H4         17

 BOND    =        1.1392  ANGLE   =      107.7352  DIHED      =       51.6193
 VDWAALS =      -50.8778  EEL     =     1336.3648  EGB        =     -132.0657
 1-4 VDW =        5.2518  1-4 EEL =    -2131.8909  RESTRAINT  =        0.0164
 EAMBER  =     -812.7239
 NMR restraints: Bond =    0.012   Angle =     0.003   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   3000      -8.1271E+02     4.7173E-01     1.6114E+00     H4         17

 BOND    =        1.1393  ANGLE   =      107.7352  DIHED      =       51.6193
 VDWAALS =      -50.8780  EEL     =     1336.3667  EGB        =     -132.0661
 1-4 VDW =        5.2519  1-4 EEL =    -2131.8922  RESTRAINT  =        0.0165
 EAMBER  =     -812.7239
 NMR restraints: Bond =    0.012   Angle =     0.003   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   3500      -8.1271E+02     4.7172E-01     1.6114E+00     H4         17

 BOND    =        1.1394  ANGLE   =      107.7353  DIHED      =       51.6192
 VDWAALS =      -50.8782  EEL     =     1336.3686  EGB        =     -132.0665
 1-4 VDW =        5.2520  1-4 EEL =    -2131.8937  RESTRAINT  =        0.0165
 EAMBER  =     -812.7239
 NMR restraints: Bond =    0.012   Angle =     0.003   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   4000      -8.1271E+02     4.7172E-01     1.6114E+00     H4         17

 BOND    =        1.1395  ANGLE   =      107.7353  DIHED      =       51.6191
 VDWAALS =      -50.8784  EEL     =     1336.3705  EGB        =     -132.0669
 1-4 VDW =        5.2522  1-4 EEL =    -2131.8951  RESTRAINT  =        0.0165
 EAMBER  =     -812.7239
 NMR restraints: Bond =    0.012   Angle =     0.003   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   4500      -8.1271E+02     4.7172E-01     1.6114E+00     H4         17

 BOND    =        1.1396  ANGLE   =      107.7354  DIHED      =       51.6191
 VDWAALS =      -50.8786  EEL     =     1336.3724  EGB        =     -132.0674
 1-4 VDW =        5.2523  1-4 EEL =    -2131.8966  RESTRAINT  =        0.0165
 EAMBER  =     -812.7239
 NMR restraints: Bond =    0.012   Angle =     0.003   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.1271E+02     4.7171E-01     1.6114E+00     H4         17

 BOND    =        1.1396  ANGLE   =      107.7354  DIHED      =       51.6190
 VDWAALS =      -50.8788  EEL     =     1336.3742  EGB        =     -132.0678
 1-4 VDW =        5.2524  1-4 EEL =    -2131.8979  RESTRAINT  =        0.0165
 EAMBER  =     -812.7239
 NMR restraints: Bond =    0.012   Angle =     0.003   Torsion =     0.000
===============================================================================


  Maximum number of minimization cycles reached.


                    FINAL RESULTS



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.1271E+02     4.7171E-01     1.6114E+00     H4         17

 BOND    =        1.1396  ANGLE   =      107.7354  DIHED      =       51.6190
 VDWAALS =      -50.8788  EEL     =     1336.3742  EGB        =     -132.0678
 1-4 VDW =        5.2524  1-4 EEL =    -2131.8979  RESTRAINT  =        0.0165
 EAMBER  =     -812.7239
 NMR restraints: Bond =    0.012   Angle =     0.003   Torsion =     0.000
===============================================================================
 ------------------------------------------------------------------------------


 Final Restraint Analysis for coords: minimize.rst7                           


 Restraints, deviations, and energy contributions:    pencut =   -1.00

 ------------------------------------------------------------------------------
     First atom        Last atom    curr. value target deviation  penalty
 ------------------------------------------------------------------------------
  DUM  DM1    3 --  C7   CB6    1:    9.305    9.302    0.003    0.000 d    0: 0
  DUM  DM2    4 --  C7   CB6    1:  147.379  147.398    0.019    0.000 a
  DUM  DM2    4 --  DUM  DM1    3:   49.467   49.483    0.016    0.000 t
  DUM  DM1    3 --  C31  CB6    1:   62.023   62.003    0.020    0.000 a
  DUM  DM1    3 --  C7   CB6    1: -146.166 -146.178    0.012    0.000 t
  C7   CB6    1 --  C31  CB6    1:  -72.548  -72.548    0.000    0.000 t
  DUM  DM1    3 --  C    BUT    2:   12.304   12.353    0.049    0.012 d    0: 0
  DUM  DM2    4 --  C    BUT    2:  179.939  180.000    0.061    0.000 a
  DUM  DM1    3 --  C3   BUT    2:  179.690  180.000    0.310    0.003 a
                                       Total distance penalty:      0.012
                                       Total angle    penalty:      0.003
                                       Total torsion  penalty:      0.000
|                               RMS deviation from ideal bonds :      0.0094
|                               RMS deviation from ideal angles:      4.471
 ------------------------------------------------------------------------------

--------------------------------------------------------------------------------
   5.  TIMINGS
--------------------------------------------------------------------------------

|                Calc gb radii              2.18 (21.71% of Gen B)
|                Calc gb diag               3.92 (39.07% of Gen B)
|                Calc gb off-diag           3.93 (39.20% of Gen B)
|                Other                      0.00 ( 0.02% of Gen B)
|             Gen Born time             10.03 (100.0% of Nonbo)
|          Nonbond force             10.04 (90.44% of Force)
|          Bond/Angle/Dihedral        1.03 ( 9.24% of Force)
|          Other                      0.03 ( 0.31% of Force)
|       Force time                11.10 (100.0% of Runmd)
|    Runmd Time                11.10 (99.80% of Total)
|    Other                      0.02 ( 0.19% of Total)
| Total time                11.12 (100.0% of ALL  )

| Number of list builds   :          0

| Highest rstack allocated:          0
| Highest istack allocated:          0
|           Job began  at 18:22:08.610  on 04/10/2019
|           Setup done at 18:22:08.615  on 04/10/2019
|           Run   done at 18:22:19.727  on 04/10/2019
|     wallclock() was called   90028 times

|3D-RISM memory allocation summary
|Type          Maximum        Current   
|Integer       0.00000 GB     0.00000 GB
|Real          0.00000 GB     0.00000 GB
|Logical       0.00000 GB     0.00000 GB
|Character     0.00000 GB     0.00000 GB
|---------------------------------------
|Total         0.00000 GB     0.00000 GB
